__tests__/components/editor/GroupChoiceModal.test.js

```diff
@@ -91,6 +92,7 @@ describe('<GroupChoiceModal />', () => {
     it('attempts to save the RDF content with group choice when save is clicked and closes the modal', () => {
       wrapper.find('.btn-primary', { text: 'Save' }).simulate('click')
       expect(server.publishRDFResource).toHaveBeenCalled()
```

Comment:
I think you can remove this line as you're testing the same + more on the following line.

Reply:
Removed.